### PR TITLE
[ZH] Add `Background`

### PR DESCRIPTION
**Pull request by QTPC**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Beatmap/Background/zh.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
+stub: true
+tags:
+  - bg
+  - 背景
+---
+
+# 背景
+
+*关于背景的规定，详见：[审核标准](/wiki/Ranking_Criteria)*
```
Comment (RushFTK) [inline wiki/Beatmap/Background/zh.md:10]:
```suggestion
*关于背景的规定，请参阅：[谱面上架标准](/wiki/Ranking_Criteria)*
```
与[相关条目](https://github.com/ppy/osu-wiki/blob/master/wiki/Ranking_Criteria/zh.md)的翻译保持一致。

File: wiki/Beatmap/Background/zh.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+**背景** (***BG***) 图片和视频能被添加在 [谱面](/wiki/Beatmap) 之中。当以 "背景" 或 "bg" 称呼时，通常是指背景图片。
```
Comment (RushFTK) [inline wiki/Beatmap/Background/zh.md:12]:
```suggestion
**背景** (***BG***) 图片和视频能被添加在 [谱面](/wiki/Beatmap) 之中。仅写作 "背景" 或 "bg" 时，往往指的是背景图片。
```

*only*具有明显意义，翻译不应省去

File: wiki/Beatmap/Background/zh.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+
+背景会在谱面卡片，[歌曲选择界面](/wiki/Client/Interface#选歌界面) 以及游戏中显示。
```
Comment (RushFTK) [inline wiki/Beatmap/Background/zh.md:14]:
```suggestion
背景会在谱面信息卡，[歌曲选择界面](/wiki/Client/Interface#选歌界面) 以及游戏中显示。
```

我不清楚*beatmap cards*在哪里出现应该翻译成什么，所以这个review是可选的。

**Review by RushFTK — CHANGES_REQUESTED** [review]
Some of review **must be** fixed.

**Review by RushFTK — APPROVED** [review]

File: wiki/Beatmap/Background/zh.md [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+tags:
+  - bg
+  - 背景
+---
+
+# 背景
+
+*关于背景的规定，请参阅：[谱面上架标准](/wiki/Ranking_Criteria)*
+
+**背景** (***BG***) 图片和视频能被添加在 [谱面](/wiki/Beatmap) 之中。仅写作 "背景" 或 "bg" 时，往往指的是背景图片。
```
Comment (Kotoki1337) [inline wiki/Beatmap/Background/zh.md:12]:
```suggestion
**背景**（***BG***）图片和视频能被添加在 [谱面](/wiki/Beatmap) 之中。仅写作“背景”或“bg”时，往往指的是背景图片。
```
使用中文标点符号。

**Review by Kotoki1337 — CHANGES_REQUESTED** [review]

**Review by TicClick — APPROVED** [review]

**Pull request merged**